src/components/SharedComponents/DateDisplay.js

```diff
@@ -37,40 +52,37 @@ const DateDisplay = ( {
     TextComponent = Body4;
   }
 
-  const obscuredDate = geoprivacy === "obscured"
+  const dateObscured = geoprivacy === "obscured"
     || taxonGeoprivacy === "obscured"
     || geoprivacy === "private"
     || taxonGeoprivacy === "private";
 
-  const formatDate = useMemo( () => {
-    if ( !belongsToCurrentUser && obscuredDate ) {
+  const formattedDate = useMemo( () => {
+    if ( !belongsToCurrentUser && dateObscured ) {
       return formatMonthYearDate( dateString, i18n );
     }
-    return formatApiDatetime( dateString, i18n );
+    if ( asDifference ) {
```

Comment:
First I thought this needs to be the first check because if we want to force to show a difference instead of a date what does it matter if the item is obscured. But apparently on the website we also don't show the precise date of an identification or comment on an obscured observation, so all good.